modules/benchmark/DESCRIPTION

```diff
@@ -15,26 +15,28 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
     streamline the interaction between data and models, and to improve the
     efficacy of scientific investigation.
 Imports:
+    dbplyr,
+    dplyr,
+    ggplot2,
+    gridExtra,
+    lubridate (>= 1.6.0),
+    magrittr,
+    ncdf4 (>= 1.15),
     PEcAn.DB,
     PEcAn.logger,
     PEcAn.remote,
     PEcAn.settings,
     PEcAn.utils,
-    lubridate (>= 1.6.0),
-    ncdf4 (>= 1.15),
-    udunits2 (>= 0.11),
-    XML (>= 3.98-1.4),
-    dplyr,
-    ggplot2,
-    gridExtra,
     reshape2,
-    dbplyr,
     SimilarityMeasures,
-    zoo,
-    stringr
+    stringr,
```

Comment:
```suggestion
    stringr,
    tidyselect,
```

Reply:
@infotroph ill get on this ASAP. also, the pleasure was mine, got to learn so much from you guys. thanks for being patient with me hehe.

Reply:
also please suggest any error fix that I can start working on after this.